src/commonMain/kotlin/com/algolia/search/endpoint/EndpointPlaces.kt

```diff
@@ -0,0 +1,43 @@
+package com.algolia.search.endpoint
+
+import com.algolia.search.model.ObjectID
+import com.algolia.search.model.places.PlaceLanguages
+import com.algolia.search.model.places.PlacesQuery
+import com.algolia.search.model.response.ResponseSearchPlacesMono
+import com.algolia.search.model.response.ResponseSearchPlacesMulti
+import com.algolia.search.model.search.Point
+import com.algolia.search.model.search.Language
+import com.algolia.search.transport.RequestOptions
+
+
+public interface EndpointPlaces {
+
+    suspend fun searchPlaces(
+        query: PlacesQuery = PlacesQuery(),
+        requestOptions: RequestOptions? = null
+    ): ResponseSearchPlacesMulti
+
+    suspend fun searchPlaces(
+        language: Language,
```

Comment:
Why overload instead of single method with `language: Language? = null`?

Reply:
The Places search endpoint has a quirk: If you pass a `language` parameter, the JSON response will have a completely different structure.
So I need two separate methods to be able to parse the two different responses accordingly.

Reply:
Ah right :confused: